packages/cli/package.json

```diff
@@ -86,6 +86,7 @@
     "@types/write-json-file": "2.2.1",
     "@types/yauzl-promise": "2.1.0",
     "@vercel-internals/constants": "1.0.2",
+    "@vercel-internals/get-help-output": "1.0.0",
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏remove

```suggestion
```